neocore/Cryptography/Crypto.py

```diff
@@ -150,7 +149,7 @@ def VerifySignature(message, signature, public_key, unhex=True):
             try:
                 message = binascii.unhexlify(message)
             except Exception as e:
-                logger.error("could not get m: %s" % e)
+                raise MessageError(f"Could not get message: {e}")
```

Comment:
1) I think we should capture `binascii.Error` instead of `Exception`. 
2) This is a case where we ask data to be unhexlified into a `bytes`/`bytearray` object and that fails for whatever reason. This is one of those times where I think we can suppress the error and let the remainder of the code work it out to return a failure or not. It could just be a user error where the data is already in `b'\xAA\xBB'` format so to say, and they should have called the function with `unhex=False` instead of the default `unhex=True`. I don't feel confident in raising a hard exception as I simply don't know if that will break something. I think ignoring is the safest bet , even though I don't really like it.

I think the following would do
```
try:
   message = binascii.unhexlify(message)
except binascii.Error:
   pass
```

or
```
from contextlib import suppress

with suppress(binascii.Error):
   message = binascii.unhexlify(message)
```

Reply:
Okay, I understand. I am leaning toward
```
try:
   message = binascii.unhexlify(message)
except binascii.Error:
   pass
```